smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChat.java

```diff
@@ -1063,6 +1064,52 @@ public void invite(MessageBuilder messageBuilder, EntityBareJid user, String rea
         connection.sendStanza(message);
     }
 
+    /**
+     * Invites another user to the room in which one is an occupant. In contrast
+     * to the method "invite", the invitation is sent directly to the user rather
+     * than via the chat room.  This is useful when the user being invited is
+     * offline, as otherwise the invitation would be dropped.
+     *
+     * @param address the user to send the invitation to
+     * @throws NotConnectedException if the XMPP connection is not connected.
+     * @throws InterruptedException if the calling thread was interrupted.
+     */
+    public void inviteDirectly(EntityBareJid address) throws NotConnectedException, InterruptedException {
+        inviteDirectly(address, null, null, false, null);
+    }
+
+     /**
+     * Invites another user to the room in which one is an occupant. In contrast
+     * to the method "invite", the invitation is sent directly to the user rather
+     * than via the chat room.  This is useful when the user being invited is
+     * offline, as otherwise the invitation would be dropped.
+     *
+     * @param address the user to send the invitation to
+     * @param reason the purpose for the invitation
+     * @param password specifies a password needed for entry
+     * @param continue_bool specifies if the groupchat room continues a one-to-one chat having the designated thread
+     * @param thread the thread to continue
+     * @throws NotConnectedException if the XMPP connection is not connected.
+     * @throws InterruptedException if the calling thread was interrupted.
+     */
+    public void inviteDirectly(EntityBareJid address, String reason, String password, boolean continue_bool, String thread)
```

Comment:
Smack uses camel case, as most of the Java ecosystem, not snake case. I also think we do not need to encode the type in the parameter name.
```suggestion
    public void inviteDirectly(EntityBareJid address, String reason, String password, boolean continue, String thread)
```

Reply:
`continue` is a Java keyword, so I'd rather not just call it that. Have amended to `continueAsOneToOne`

Reply:
+1, but maybe go all the way and use `continueAsOneToOneChat`.

Reply:
Yeh I'd gone back and done that on second thoughts 👍